.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -6,14 +6,33 @@ labels: bug, 0 - new
 assignees: ""
 ---
 
-### Summary
+<!--
+* Before submitting an issue, please look at existing issues on https://github.com/Esri/calcite-components/issues
+If someone has already opened an issue for what you are experiencing,
+please add a 👍 reaction to the existing issue instead of creating a new one.
+
+* For support requests, please go to https://developers.arcgis.com/calcite-design-system/community/
+
+* Note that this issue will be closed if necessary information is missing.
+-->
```

Comment:
nitpick, can we use this wording?
```markdown
<!--
* Before submitting an issue, please check to see if the issue already exists: https://github.com/Esri/calcite-components/issues
If someone has already opened an issue for what you are experiencing,
please add a 👍 reaction to the existing issue instead of creating a new one.

* For additional support, please check: https://developers.arcgis.com/calcite-design-system/community/

* Note that this issue will be closed if the required information below is missing.
-->
```